strings/french.strings

```diff
@@ -1,205 +1,205 @@
-"Group members" = "Group members";
+/* views/containers/dashboard/GroupMembersAllModal.vue */
+"Group members" = "Membres du groupe";
 
 /* views/pages/Home.vue */
 "Logout" = "Déconnexion";
 
-/* MISSING TRANSLATION - views/components/AvatarUpload.vue */
-"Failed to save avatar. {reportError}" = "Failed to save avatar. {reportError}";
+/* views/components/AvatarUpload.vue */
+"Failed to save avatar. {reportError}" = "L'enregistrement de l'avatar a échoué. {reportError}";
 
 /* views/containers/proposals/ProposalVoteOptions.vue */
-"Are you sure you want to cancel this proposal?" = "Êtes-vous certain de vouloir annuler cette offre?";
+"Are you sure you want to cancel this proposal?" = "Êtes-vous sûr(e) de vouloir annuler cette proposition ?";
 
-/* MISSING TRANSLATION - views/containers/user-settings/UserProfile.vue, views/pages/GroupSettings.vue */
-"Your changes were saved!" = "Your changes were saved!";
+/* views/containers/user-settings/UserProfile.vue, views/pages/GroupSettings.vue */
+"Your changes were saved!" = "Vos modifications ont été enregistrées !";
 
-/* MISSING TRANSLATION - views/containers/access/LoginModal.vue, views/pages/Join.vue */
-"Not on Group Income yet?" = "Not on Group Income yet?";
+/* views/containers/access/LoginModal.vue, views/pages/Join.vue */
+"Not on Group Income yet?" = "Pas encore sur Group Income ?";
 
 /* MISSING TRANSLATION - views/containers/user-settings/Troubleshooting.vue */
-"Re-sync failed. {reportError}" = "Re-sync failed. {reportError}";
+"Re-sync failed. {reportError}" = "L'actualisation a échoué. {reportError}";
 
-/* MISSING TRANSLATION - views/containers/contributions/ContributionsWidget.vue */
-"You will receive {amount}." = "You will receive {amount}.";
+/* views/containers/contributions/ContributionsWidget.vue */
+"You will receive {amount}." = "Vous recevrez {amount}.";
 
-/* MISSING TRANSLATION - views/containers/contributions/Contribution.vue */
-"Parties" = "Parties";
+/* views/containers/contributions/Contribution.vue */
+"Parties" = "Fêtes";
 
 /* views/containers/access/SignupModal.vue */
 "Sign Up" = "Créer un compte";
 
 /* MISSING TRANSLATION - views/containers/proposals/ProposalBox.vue */
-"{username} is proposing:" = "{username} is proposing:";
+"{username} is proposing:" = "{username} propose :";
 
 /* views/containers/access/LoginForm.vue, views/pages/Home.vue */
 "Login" = "Se connecter";
 
-/* MISSING TRANSLATION - views/containers/group-settings/GroupLeaveModal.vue, views/pages/GroupSettings.vue */
-"Leave group" = "Leave group";
+/* views/containers/group-settings/GroupLeaveModal.vue, views/pages/GroupSettings.vue */
+"Leave group" = "Quitter le groupe";
 
-/* MISSING TRANSLATION - views/containers/group-settings/InvitationsTable.vue */
-"{hours}h {minutes}m left" = "{hours}h {minutes}m left";
+/* views/containers/group-settings/InvitationsTable.vue */
+"{hours}h {minutes}m left" = "{hours}h {minutes}min restantes";
```

Comment:
I agree this is inconsistent with the current English strings, but I would prefer not to use `m` because it actually stands for the meter, while the legal symbol of the minute is `min`:

According to https://fr.wikipedia.org/wiki/Minute_(temps):
> Le symbole de la minute temporelle est « min » (invariable, sans point). Le Système international d'unités (SI) n'admet ni le symbole « m », qu'il utilise pour le mètre, ni « mn »2.
>
> Depuis le décret no 75-1200 du 23 décembre 19753, le symbole min est le seul légal en France 

So for consistency I was tempted to update the English strings to use `min` as well, but I was not sure that was on-topic for a translation.

Reply:
ok